app/routes.py

```diff
@@ -1,2 +1,272 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, make_response, request
+from app.models.task import Task
+from app.models.goal import Goal
+from app import db
+from datetime import datetime, timezone
+import requests
+import os
 
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+def is_input_valid(number):
+    try:
+        int(number)
+    except:
+        return make_response(f"{number} is not an int!", 400)
+
+
+def is_parameter_found(model, parameter_id):
+    if is_input_valid(parameter_id):
+        return is_input_valid(parameter_id)
+    elif model.query.get(parameter_id) is None:
+        return make_response(f"{parameter_id} was not found!", 404)
+    
+
```

Comment:
Suggestion for refactoring `is_parameter_found`

def get_task:
     #check that it's valid using is_input_valid
         abort if not
      #check that it's found
         abort if not (using get_or_404)
       return task